src/main/java/com/thullo/service/BoardServiceImpl.java

```diff
@@ -51,10 +53,19 @@ public BoardResponse createBoard(BoardRequest boardRequest, UserPrincipal userPr
         if (Helper.isNullOrEmpty(boardRequest.getName())) throw new BadRequestException("Board name cannot be empty");
         User user = findByEmail(userPrincipal.getEmail());
         Board board = mapper.map(boardRequest, Board.class);
-        board.setUser(user);
+
+        if(boardRequest.getBoardVisibility() == null){
```

Comment:
Do this check inside `getBoardVisibility` return `PRIVATE` if `boardRequest.getBoardVisibility()` is `null`